### DISCUSSION TITLE
Change Password

### DISCUSSION THREAD

**Discussion by kartikpatel211** [soto-project/soto-cognito-authentication-kit#16] [General]
Hi, I am wondering if library has change password feature - once user login and want to change password. I have searched for it but not found anything related to that. If this is already implemented please guide. If not then how can we to manage change password feature using this library? Thanks.

**Comment by adam-fowler** [comment]
There is no direct implementation of change password in this library, but you could use the `changePassword` function that is available in `SotoCognitoIdentityProvider`
